helm_deploy/values-dev.yaml

```diff
@@ -14,6 +14,7 @@ generic-service:
     SPRING_PROFILES_ACTIVE: "stdout"
     HMPPS_AUTH_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
     PRISON_API_URL: https://api-dev.prison.service.justice.gov.uk
+    PRISONER-OFFENDER-SEARCH_URL: https://prisoner-offender-search-dev.prison.service.justice.gov.uk
```

Comment:
fussy I know but please use _ rather than - (PRISONER-OFFENDER-SEARCH_URL) unless there is a particular reason for it @Value("\${"PRISONER-OFFENDER-SEARCH.URL"}") it's just because we have not done it before... I like consistency.. it's ether one way or the other not a mix.

Reply:
updated the application.yml to 
prisoner:
  offender:
    search:

and the values.yaml to PRISONER_OFFENDER_SEARCH_URL as suggested.